--- plugins/views_plugin_display.inc
+++ plugins/views_plugin_display.inc
@@ -291,30 +291,6 @@ class views_plugin_display extends views_plugin {
     }
   }
 
-  /**
-   * Set default options.
-   *
-   * Displays put their options in a different place than everything else; also
-   * displays spread their options out. We don't want to set defaults for
-   * items that are normally defaulted elsewhere.
-   */
-  function _set_option_defaults(&$storage, $options, $level = 0) {
-    foreach ($options as $option => $definition) {
-      // If defaulted to elsewhere and we're not the default display, skip.
-      if ($level == 0 && !$this->is_default_display() && !empty($options['defaults']['default'][$option])) {
-        continue;
-      }
-
-      if (isset($definition['contains']) && is_array($definition['contains'])) {
-        $storage[$option] = array();
-        $this->_set_option_defaults($storage[$option], $definition['contains'], $level++);
-      }
-      else {
-        $storage[$option] = isset($definition['default']) ? $definition['default'] : NULL;
-      }
-    }
-  }
-
   function option_definition() {
     $options = array(
       'defaults' => array(
